examples/jsm/tsl/display/GTAONode.js

```diff
@@ -107,7 +110,22 @@ class GTAONode extends TempNode {
 
 		const uvNode = uv();
 
-		const sampleDepth = ( uv ) => this.depthNode.uv( uv ).x;
+		const sampleDepth = ( uv ) => {
+
+			const depth = this.depthNode.uv( uv ).x;
+
+			if ( builder.renderer.logarithmicDepthBuffer === true ) {
+
+				const viewZ = logarithmicDepthToViewZ( depth, this.cameraNear, this.cameraFar );
+
+				return viewZToPerspectiveDepth( viewZ, this.cameraNear, this.cameraFar );
```

Comment:
Can we check here if the camera is orthographic and then use `viewZToOrthographicDepth()` instead?